server/app/boot.rb

```diff
@@ -17,17 +17,16 @@
 require 'tilt/jbuilder.rb'
 require 'mongoid/enum'
 
-Dir[__dir__ + '/initializers/*.rb'].sort.each {|file| require file }
-
-Dir[__dir__ + '/authorizers/*.rb'].each {|file| require file }
-
-Dir[__dir__ + '/models/*.rb'].each {|file| require file }
-
-Dir[__dir__ + '/helpers/*.rb'].each {|file| require file }
-
-Dir[__dir__ + '/mutations/**/*.rb'].each {|file| require file }
-
-Dir[__dir__ + '/jobs/**/*.rb'].each {|file| require file }
-
-Dir[__dir__ + '/services/**/*.rb'].each {|file| require file }
+def require_glob(glob)
```

Comment:
This root level method only gets defined if the program is started through loading boot.rb